sources/api/netdog/src/cli/check_net_config.rs

```diff
@@ -0,0 +1,38 @@
+use super::{check_net_config, error, Result};
+use argh::FromArgs;
+use snafu::OptionExt;
+
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "check-net-config")]
```

Comment:
Yeah, I'm fine with that. I've chosen 3 different names because I couldn't decide which one I liked the most. Validate does seem to be the wording we are settling on so I'll make the code consistent.